pkg/backup/backup/backup_manager.go

```diff
@@ -846,6 +855,15 @@ func (bm *backupManager) makeBRBackupJob(backup *v1alpha1.Backup) (*batchv1.Job,
 		},
 	}
 
+	// for volume backup initializing job, we should set resource requirement empty
+	// avoid it consuming too much resource
+	if backup.Spec.Mode == v1alpha1.BackupModeVolumeSnapshot &&
+		backup.Spec.FederalVolumeBackupPhase == v1alpha1.FederalVolumeBackupInitialize {
+		bm.setBackupPodResourceRequirementsEmpty(&job.Spec.Template)
+		// for volume backup initializing job, set deadline of the job in case it blocks GC and pd schedule indefinitely
+		job.Spec.ActiveDeadlineSeconds = pointer.Int64(int64(volumeBackupInitJobMaxActiveSeconds))
```

Comment:
@WangLe1321 this gave me compile error `pkg/backup/backup/backup_manager.go:864:44: undefined: pointer.Int64` should we use pointer.Int64Ptr instead?